resolwe_bio/migrations/0011_nucletide_seq.py

```diff
@@ -0,0 +1,110 @@
+import gzip
+import os
+import shutil
+
+from django.conf import settings
+from django.db import migrations
+
+from resolwe.flow.migration_ops import DataDefaultOperation, ResolweProcessAddField, ResolweProcessRenameField
+from resolwe.flow.utils import iterate_fields
+
+FASTA_SCHEMA = {
+    'name': 'fasta',
+    'label': 'FASTA file',
+    'type': 'basic:file:',
+}
+
+
+class DefaultUnzipFasta(DataDefaultOperation):
+    """Set default value."""
+
+    def prepare(self, data, from_state):
+        pass
+
+    def get_default_for(self, data, from_state):
+        """Return default for given data object."""
+        fastagz = os.path.join(
```

Comment:
Directory path is no longer bound to the data id, but to the location id. You should use [`data.location.get_path()`](https://github.com/genialis/resolwe/blob/38f5e9a1bc45b3cdcc16f737f44553a1d10d21a1/resolwe/flow/models/data.py#L584) or equivalent.